src/mediumComplexityProblem7/SortArray.java

```diff
@@ -0,0 +1,39 @@
+package mediumComplexityProblem7;
+
+public class SortArray {
+    public void ascendingSort(int[] inputArray) {
+        // Sort elements of original array in ascending order
+        int firstCounter;
+        for (firstCounter = 0; firstCounter < inputArray.length; firstCounter++) {
+            for (int secondCounter = firstCounter + 1; secondCounter < inputArray.length; secondCounter++) {
+                if (inputArray[firstCounter] > inputArray[secondCounter]) {
+                    int temp = inputArray[firstCounter];
+                    inputArray[firstCounter] = inputArray[secondCounter];
+                    inputArray[secondCounter] = temp;
+                }
+            }
+        }
+        // Remove duplicates element from Ascending sort Array
+        int checkDuplicateCounter;
+        int tempArrayCounter = 0;
+        for (checkDuplicateCounter = 0; checkDuplicateCounter < inputArray.length - 1; checkDuplicateCounter++) {
+            if (inputArray[checkDuplicateCounter] != inputArray[checkDuplicateCounter + 1]) {
+                inputArray[tempArrayCounter] = inputArray[checkDuplicateCounter];
```

Comment:
why require the second loop. try to remove duplicate in first loop only.

Reply:
Done!! Added code to sort and remove duplicate elements from single loop